src/data/localFont.js

```diff
@@ -0,0 +1,37 @@
+import localFont from "next/font/local";
+
+const nohemi = localFont({
+  src: [
+    {
+      path: "../../public/assets/font/Nohemi-Black-BF6438cc565e67b.woff",
```

Comment:
Using `@/public/assets` feels shorter than `../../` could you try that?

Reply:
Yes i can, on it.